inst/qml/uniDimBayesian.qml

```diff
@@ -231,6 +231,7 @@ Form
 				{
 					DoubleField
```

Comment:
In this DoubleField, you should set:
`max: probTableValueHigh.value`

where `probTableValueHigh` is the (new) id of the second DoubleField.

Reply:
I have tried that, but it seems as though when the lower bound field is evaluated it does not see a higher value for the upper bound: 
<img width="331" alt="Screenshot 2021-08-11 at 17 28 51" src="https://user-images.githubusercontent.com/38500953/129058302-c2b3258b-9f9d-47fb-837e-bcecf5e7d4c7.png">